VideoView.js

```diff
@@ -0,0 +1,11 @@
+VideoView = Backbone.View.extend({
+  el: $('#video-posts'),
+
+  template: Handlebars.compile($('#video-sidebar').html()),
+
+  render: function () {
+    this.$el.html(this.template(this.model.toJSON()));
```

Comment:
I mean you rendered this view totally right, but then drove off a cliff on AppView?  Why would you do that?  If it's not broken, don't try to fix it (or come up with a new creative way to do it).